vspace/src/main/java/edu/asu/diging/vspace/core/data/SequenceRepository.java

```diff
@@ -18,5 +19,7 @@
     
     @Query("SELECT d FROM Sequence d WHERE d.module.id = ?1 and d.id = ?2")
     public Sequence findSequenceForModuleAndSequence(String moduleId, String sequenceId);
-
+    
+    @Query(value="SELECT d.startSequence_id FROM Module d where d.startSequence_id IS NOT NULL", nativeQuery = true)
```

Comment:
is there a reason this needs to be a native query?

Reply:
Changed the native query to jpql query.